package.json

```diff
@@ -67,8 +67,8 @@
     "clean": "rm -rf node_package/lib",
     "start": "nps",
     "prepare": "yarn run build",
-    "prepublish": "npm run prepare",
-    "build": "yarn run clean && yarn run tsc",
+    "prepublishOnly": "npm run prepare",
```

Comment:
This change isn't relevant to typescript, but stops yarn from running `yarn build` twice after `yarn install`.